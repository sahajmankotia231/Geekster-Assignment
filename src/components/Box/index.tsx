import styled from "styled-components";

import { WHITE } from "constants/colors";
import { DEFAULT_RADIUS } from "constants/styles";

const Box = styled.div`
  background-color: ${WHITE};
  border-radius: ${DEFAULT_RADIUS}px;
  box-shadow: 0px 0px 1px rgba(68, 80, 95, 0.08),
    0px 1px 4px rgba(68, 80, 95, 0.2);
  padding: 12px 8px;
`;

export default Box;
