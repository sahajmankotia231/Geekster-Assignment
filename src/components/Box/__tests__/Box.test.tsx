import React from "react";
import { render } from "@testing-library/react";

import Box from "..";

describe("<Box />", () => {
  it("should render properly", () => {
    const { container } = render(<Box>Box content</Box>);
    expect(container.firstChild).toMatchSnapshot();
  });
});
