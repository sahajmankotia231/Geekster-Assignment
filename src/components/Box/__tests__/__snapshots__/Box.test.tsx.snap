// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Box /> should render properly 1`] = `
<div
  class="sc-bdvvaa dfsOak"
>
  Box content
</div>
`;
